src/appleseed/foundation/image/accumulatortile.cpp

```diff
@@ -0,0 +1,98 @@
+
+//
+// This source file is part of appleseed.
+// Visit https://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
+// Copyright (c) 2014-2018 Francois Beaune, The appleseedhq Organization
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+// Interface header.
+#include "accumulatortile.h"
+
+// appleseed.foundation headers.
+#include "foundation/image/pixel.h"
+#include "foundation/math/scalar.h"
+#include "foundation/platform/atomic.h"
+
+namespace foundation
+{
+
+AccumulatorTile::AccumulatorTile(
+    const size_t            width,
+    const size_t            height,
+    const size_t            channel_count)
+  : Tile(width, height, channel_count + 1, PixelFormatFloat)
+  , m_crop_window(Vector2u(0, 0), Vector2u(width - 1, height - 1))
+{
```

Comment:
Ether you add a blank line here or you suppress the blank line at the beginning of the namespace. I don't remember what the guideline is.

Reply:
We don't have a strict rule on this. Just be consistent until we switch to clang-format.